.github/workflows/main.yaml

```
@@ -30,3 +33,44 @@ jobs:
+          git push origin "refs/tags/$TAG" --force
+      - name: Create Release
+        uses: softprops/action-gh-release@v2
+        with:
+          target_commitish: ${{ github.sha }}
+          token: ${{ secrets.PAT }}
+          tag_name: ${{ steps.extract_version.outputs.tag_name }}
+          prerelease: false
+          make_latest: false
+          draft: false
+          generate_release_notes: true
+          overwrite_existing: true
```

actionlint (1.7.12)

[error] input "overwrite_existing" is not defined in action "softprops/action-gh-release@v2". available inputs are "append_body", "body", "body_path", "discussion_category_name", "draft", "fail_on_unmatched_files", "files", "generate_release_notes", "make_latest", "name", "overwrite_files", "prerelease", "preserve_order", "previous_tag", "repository", "tag_name", "target_commitish", "token", "working_directory"

(action)